airgun/views/common.py

```diff
@@ -84,12 +99,13 @@ class resources(AddRemoveResourcesView):
     class ListRemoveTab(SatSecondaryTab):
         TAB_NAME = 'List/Remove'
         searchbox = Search()
-        remove_button = Text('.//button[@ng-click="removeSelected()"]')
-        table = Table(locator=".//table")
-        no_rows = Text(
-            ".//table//span[@data-block='no-rows-message' or "
-            "@data-block='no-search-results-message']"
+        remove_button = Text(
+            './/div[@data-block="list-actions"]'
+            '//button[contains(@ng-click, "remove")]'
         )
+        table = Table(locator=".//table")
+        columns_exists = Text(
+            ".//table//*[contains(name(), 'body')]/tr[1]/td[2]")
```

Comment:
Hm... When we previously discussed this, you've rejected proposal to use indexes and described it's a bad idea here. Why changed your mind?

Reply:
Unless I'm missing something, this is screaming "give me an ID I can use"...

Reply:
I changed my mind, because previous approach stopped to work in other places, this one is only one that will ever work

And no, we do not need id here as we use that xpath to calculate amount of columns in the table, not to address the element

Reply:
If table is empty - it contains span with `@data-block='no-rows-message'` or `@data-block='no-search-results-message'`, that's basically good enough "id" for us, no updates needed. And if table is not empty - we probably don't need such id (where to place it? first cell of the table will contain id like 'im_not_empty_table'? doesn't seem right). My question was rather - if table is not empty, we could check that span with `data-block='no-search-results-message'` is absent, instead of trying to access specific cell to see whether there's content.

Reply:
>because previous approach stopped to work in other places, this one is only one that will ever work

@oshtaier i think that's what @omaciel was pointing out - if there're places where no ids are present for "empty" message, maybe we should update them?

Reply:
problem is completely in other area. Element is exist, but not visible, so `is_displayed` will not work in that case. I would say that is a bug for webUI from UX perspective, but it is so minor that it will take years before it will be fixed